src/api/vdom.js

```diff
@@ -94,8 +94,13 @@ attributes[symbols.default] = function (elem, name, value) {
     value = `[slot="${value}"]`;
   }
 
-  // Set defined props on the element directly.
-  if (name in elem) {
+  // Set defined props on the element directly. This ensures properties like
+  // "value" on <input> elements get set correctly. Setting those as attributes
+  // doesn't always work and setting props is faster than attributes.
+  //
+  // However, certain props on SVG elements are readonly and error when you try
+  // to set them.
+  if (name in elem && !('ownerSVGElement' in elem)) {
```

Comment:
Is it possible to make this fix more general by checking that the property isn't read-only, or is there some performance consideration here?